openwisp_radius/base/models.py

```diff
@@ -1034,7 +1035,14 @@ class AbstractOrganizationRadiusSettings(UUIDModel):
         null=True,
         blank=True,
         help_text=_(
-            'Additional configuration for SMS backend in JSON format, if needed'
+            'Additional configuration for SMS backend in JSON format (optional)'
```

Comment:
I think `Additional configuration for SMS backend in JSON format, if needed` -> `Additional configuration for SMS backend in JSON format (optional)` is more clear and I was creating a migration, so I added this, let me know if I should revert it.